telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/TelemetryGenerator.kt

```diff
@@ -86,7 +86,7 @@ private fun FileSpec.Builder.generateTelemetryEnumType(item: TelemetryMetricType
     item.allowedValues!!.forEach { enumValue ->
         enum.addEnumConstant(
             enumValue.toString().replace(Regex("\\s"), "_").toTypeFormat(), TypeSpec.anonymousClassBuilder()
```

Comment:
For Kotlin we converted the "key" but not the "value" to get rid of spaces. This was an oversight and testing of course caught this. 

Thanks for pushing for testing.